### PR TITLE
chore: nrwl to nx namespace

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-758

- Remove deprecated `@nrwl/tao` 
- Remove deprecated config options
- Move from `@nrwl`  to `@nx` 
- Add some `@swc` packages to align with NX defaults config

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> One day they will all understand

File: libs/shared/ui/src/lib/components/layouts/breadcrumb/breadcrumb.spec.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { render } from '__tests__/utils/setup-jest'
-// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
-// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
-// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
-// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+// eslint-disable-next-line @nx/enforce-module-boundaries
+// eslint-disable-next-line @nx/enforce-module-boundaries
+// eslint-disable-next-line @nx/enforce-module-boundaries
+// eslint-disable-next-line @nx/enforce-module-boundaries
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/layouts/breadcrumb/breadcrumb.spec.tsx:5]: Keep one comment ?

**Review by RemiBonnet — APPROVED** [review]

File: libs/shared/ui/src/lib/components/layouts/breadcrumb/breadcrumb.spec.tsx (hunk 1 above)
Comment (ctjhoa) [reply libs/shared/ui/src/lib/components/layouts/breadcrumb/breadcrumb.spec.tsx]: https://github.com/Qovery/console/pull/714/commits/b735b00abbda4bc13f5f907abd8b896280ca46a1

**Pull request merged**